src/HomeKit/HMMutablePresenceEvent.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.HomeKit {
+
+	partial class HMMutablePresenceEvent {
+
+		public HMPresenceType PresenceType {
```

Comment:
yup we can break before it becomes stable, but we can keep the binary compat since now :)